create_rental.rb

```diff
@@ -0,0 +1,12 @@
+def create_rental(books, people, rentals)
+  book = select_book(books)
+  person = select_person(people)
+
+  if book && person
+    date = rental_date
+    create_and_add_rental(rentals, date, book, person)
+    puts 'Rental created successfully'
+  else
+    puts 'Invalid input, please try again'
+  end
+end
```

Comment:
- You have done great so far :+1: But there are a few errors you still need to fix before moving to the next milestone. :+1: :smile: 
- [ ] Looking at the video below, you can see that the App crashes whenever I try to create a rental, this is not meant to be so. I suggest you look into this and fix this. Do well to thoroughly test the app before requesting another review. :+1: :100: :+1: 

- - VIDEO REFERENCE:

https://github.com/dvcorreia1984/oop-school-library/assets/107632780/3a9f7a62-f0da-4403-ad43-aa52e24ef553